### ISSUE TITLE
Preparing for CaaSP 4 regarding Helm and Tiller

### ISSUE THREAD

**Issue by gaktive** [SUSE/doc-cap#513]
CaaSP 4 exists as a beta but the team is looking at eventually replacing helm 2 with helm 3. Not sure exactly when it will land but CAP needs to prepare for it.

Helm 3 currently is an alpha release so I'm not sure exactly how primetime ready it will be. However, CAP won't currently work with Helm 3 based on various syntactical changes in the config. We're scheduling work to eventually be Helm 3 compatible.

However, for now, we will need to look over what we currently have under "5.6 Deploy with Helm" and expand that to specify that Helm 2 will be needed to install and manage CAP. Relevant for CaaSP 4, Tiller will need to be installed since it is either optional or outright removed, which may be in contrast to CaaSP 3.

**Comment by btat** [comment]
Via bsc#1142360 regarding Tiller

```
Since caasp4 the installation of helm on the server side requires a service account (like it was the case for Azure once). 

The way we managed to deploy helm to caasp4 was via these commands:


`kubectl create serviceaccount tiller --namespace kube-system`

`kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller`
￼
`helm init --upgrade --service-account tiller`

We might want to add this to our documentation.
```

**Comment by gaktive** [comment]
At this point, CaaSP 4 is sticking with Helm 2 based on how ready Helm 3 is. However, they'll keep tabs on the v3 progress for possible consideration in updated releases.

**Comment by btat** [comment]
Addressed in #524
